level-2/스킬트리&49993&.js

```diff
@@ -13,4 +13,57 @@ function solution(skill, skill_trees) {
 function checkOrder(checkArr) {
     for (let i = 0; i < checkArr.length; i++) if (checkArr[i] !== i) return false
     return true
-}
\ No newline at end of file
+}
+
+//정답 2 - minkyeongJ
+function solution(skill, skill_trees) {
+	  //return할 변수
+    var answer = 0;
+
+    //파라미터로 받은 값을 모두 비교가능하게 쪼갬
+    const skillArr = Array.from(skill);
```

Comment:
문자열은 유사배열 객체이기 때문에 위와 같은 방법도 가능합니다!
다만 함수는 최대한 적게 써야 좋으니 아래와 같이 가독성 좋은 방법도 고려해보세요!

```suggestion
    const skillArr = [...skill]
```
혹은 
`const skillArr = skill.split('')`